.Rbuildignore

```diff
@@ -9,3 +9,5 @@
 ^CRAN-RELEASE$
 ^data-raw$
```

Comment:
There's still a `.DS_Store` file in the root directory that should be ignored from .RBuild and from git, as well as deleted from the repository